packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7923,8 +7923,8 @@ export default class Meeting extends StatelessWebexPlugin {
       if (roles.includes(SELF_ROLES.COHOST)) {
         return 'cohost';
       }
-      if (roles.includes(SELF_ROLES.PRESENTER)) {
-        return 'presenter';
+      if (roles.includes(SELF_ROLES.PANELIST)) {
+        return 'panelist';
```

Comment:
suggestion: the existing jsdoc for this function's return value does not mention presenter nor panelist, maybe it would be worth updating it and just removing the listed values from the comment altogether, it's better to just specify the returning type in the code like `getCurUserType(): 'host'|'cohost'|'panelist'|'attendee'` as this is typescript. Or actually even better to use the type from the event dictionary - I think we have them defined now somewhere